pennylane_cirq/cirq_device.py

```diff
@@ -139,20 +139,26 @@ def __init__(self, wires, shots, analytic, qubits=None):
         "CZ": CirqOperation(lambda: cirq.CZ),
         "PhaseShift": CirqOperation(lambda phi: cirq.ZPowGate(exponent=phi / np.pi)),
         "CPhase": CirqOperation(lambda phi: cirq.CZPowGate(exponent=phi / np.pi)),
-        "RX": CirqOperation(cirq.rx),
-        "RY": CirqOperation(cirq.ry),
-        "RZ": CirqOperation(cirq.rz),
-        "Rot": CirqOperation(lambda a, b, c: [cirq.rz(a), cirq.ry(b), cirq.rz(c)]),
-        "CRX": CirqOperation(lambda phi: cirq.ControlledGate(cirq.rx(phi))),
-        "CRY": CirqOperation(lambda phi: cirq.ControlledGate(cirq.ry(phi))),
-        "CRZ": CirqOperation(lambda phi: cirq.ControlledGate(cirq.rz(phi))),
-        "CRot": CirqOperation(
+        "RX": CirqOperation(lambda phi: cirq.XPowGate(exponent=phi / np.pi, global_shift=-0.5)),
+        "RY": CirqOperation(lambda phi: cirq.YPowGate(exponent=phi / np.pi, global_shift=-0.5)),
+        "RZ": CirqOperation(lambda phi: cirq.ZPowGate(exponent=phi / np.pi, global_shift=-0.5)),
```

Comment:
Do these need to be replaced? They worked fine when I tested them.

Reply:
I thought they were faulty, but maybe that was miscommunication on my end, sorry!
I am kind of confused by the global phase factor...